mas-foundation/src/main/java/com/ca/mas/messaging/MASMessenger.java

```diff
@@ -48,6 +49,25 @@
     void sendMessage(MASMessage message, MASUser user, String topic, MASCallback<Void> callback);
 
     /**
+     * This method sends a message from the current user to an existing group.
```

Comment:
We should be consistent with capitalizing Message/Group/Topic/Callback in our documentation; if one of them is, then all of them should.